From: Luca Weiss <luca.weiss@fairphone.com>
Date: Mon, 21 Aug 2023 14:39:20 +0200
Subject: Input - goodix_berlin_a_driver: remove extra newlines

The ts_err macro already adds a newline after each print.
---
 drivers/input/touchscreen/goodix_berlin_a_driver/goodix_brl_hw.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/goodix_berlin_a_driver/goodix_brl_hw.c b/drivers/input/touchscreen/goodix_berlin_a_driver/goodix_brl_hw.c
index 178b491..5634756 100644
--- a/drivers/input/touchscreen/goodix_berlin_a_driver/goodix_brl_hw.c
+++ b/drivers/input/touchscreen/goodix_berlin_a_driver/goodix_brl_hw.c
@@ -239,7 +239,7 @@ ts_err("zmw---brl_power_on---op");
 			gpio_direction_output(iovdd_gpio, 1);
 		} else if (cd->iovdd) {			
 			ret = regulator_enable(cd->iovdd);
-ts_err("zmw:name=[%s] line=[%d] iovdd \n",__func__,__LINE__);			
+ts_err("zmw:name=[%s] line=[%d] iovdd",__func__,__LINE__);
 			if (ret < 0) {
 				ts_err("Failed to enable iovdd:%d", ret);
 				goto power_off;
@@ -252,7 +252,7 @@ ts_err("zmw:name=[%s] line=[%d] iovdd \n",__func__,__LINE__);
 			gpio_direction_output(avdd_gpio, 1);
 		} else if (cd->avdd) {
 			ret = regulator_enable(cd->avdd);	
-ts_err("zmw:name=[%s] line=[%d] avdd \n",__func__,__LINE__);					
+ts_err("zmw:name=[%s] line=[%d] avdd",__func__,__LINE__);
 			if (ret < 0) {
 				ts_err("Failed to enable avdd:%d", ret);
 				goto power_off;
